packages/cli/commands/sandbox/delete.js

```diff
@@ -10,7 +10,7 @@ const { trackCommandUsage } = require('../../lib/usageTracking');
 const { loadAndValidateOptions } = require('../../lib/validation');
 const { logError, debugError } = require('../../lib/errorHandlers/index');
 const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/index');
-const { deleteSandbox } = require('@hubspot/local-dev-lib/sandboxes');
+const { deleteSandbox } = require('@hubspot/local-dev-lib/api/sandboxHubs');
```

Comment:
The call is on line 141 and the return value is being ignored, that's why there are no other changes here.